tests/test_private_vlan_type.rb

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2013-2016 Cisco and/or its affiliates.
```

Comment:
As mentioned above, I would prefer you add these tests to `test_vlan.rb` or a single test file called `test_private_vlan.rb`